shared/chat/confirm-payment/index.js

```diff
@@ -0,0 +1,236 @@
+          <Kb.Text type="HeaderExtrabold" style={styles.xlmTotal}>
+            {props.xlmTotal}
+          </Kb.Text>
+          <Kb.Text type="Body" style={styles.headerText}>
+            (APPROXIMATELY {props.displayTotal})
+          </Kb.Text>
+        </Kb.Box2>
+        <Kb.ScrollView style={styles.paymentsContainer}>
+          {props.payments.map(p => (
+            <Kb.Box2 key={p.username} direction="horizontal" fullWidth={true} style={styles.paymentContainer}>
+              <PaymentRow {...p} />
+            </Kb.Box2>
+          ))}
+        </Kb.ScrollView>
+        <Kb.ButtonBar align="center" direction="row" fullWidth={true}>
+          <Kb.Button
+            type="Secondary"
+            small={Styles.isMobile}
+            onClick={props.onCancel}
+            style={styles.cancelButton}
+            label="Cancel"
+          />
+          <Kb.WaitingButton
+            style={styles.submitButton}
+            small={Styles.isMobile}
+            type="PrimaryGreen"
+            onClick={props.onAccept}
+            waitingKey={null}
+            children={
+              <Kb.Icon
+                color={Styles.globalColors.white}
+                style={Kb.iconCastPlatformStyles(styles.submitIcon)}
+                type="iconfont-stellar-send"
+              />
+            }
+            label={'Send ' + props.xlmTotal}
+          />
+        </Kb.ButtonBar>
+      </Kb.Box2>
+    )}
+  </Kb.MaybePopup>
+)
+
+const styles = Styles.styleSheetCreate({
+  buttonContainer: Styles.platformStyles({
+    common: {
+      justifyContent: 'space-between',
+    },
+  }),
+  cancelButton: Styles.platformStyles({
+    isElectron: {
+      height: 40,
+    },
+  }),
+  container: Styles.platformStyles({
+    common: {
+      backgroundColor: Styles.globalColors.white,
+    },
+    isElectron: {
+      height: 458,
+      width: 360,
+    },
+  }),
+  errorClose: {
+    padding: Styles.globalMargins.tiny,
+  },
+  errorText: {
+    color: Styles.globalColors.red,
+  },
+  fullErrorContainer: Styles.platformStyles({
+    isElectron: {
+      padding: 20,
+    },
+  }),
+  headerText: Styles.platformStyles({
+    common: {
+      color: Styles.globalColors.white,
+      fontSize: 11,
+    },
+  }),
+  icon: Styles.platformStyles({
+    isElectron: {
+      marginBottom: 16,
+      marginTop: 35,
+    },
+  }),
+  paymentContainer: Styles.platformStyles({
+    common: {
+      borderColor: Styles.globalColors.lightGrey,
+      borderStyle: 'solid',
+      borderWidth: 1,
+      justifyContent: 'space-between',
+      padding: Styles.globalMargins.tiny,
+    },
+    isElectron: {
+      margin: -1,
+    },
+  }),
+  paymentTotalsContainer: Styles.platformStyles({
+    common: {
+      alignItems: 'flex-end',
+    },
+  }),
+  paymentsContainer: Styles.platformStyles({
+    isElectron: {
+      minHeight: 150,
+    },
+  }),
+  submitButton: Styles.platformStyles({
+    isElectron: {
+      height: 40,
+      minWidth: 225,
```

Comment:
why is this needed

Reply:
So the button is the same width regardless of XLM amount

Reply:
Why not flex 1